2023-01-28 14:18:05,961 [124]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 14:18:05,963 [126]   INFO - login.py_run:30 - Starting Login Form...
2023-01-28 14:18:13,110 [127]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 14:18:13,112 [129]   INFO - login.py_run:30 - Starting Login Form...
2023-01-28 14:18:15,923 [2940]   INFO - login.py_quit:17 - Exit Pressed.
2023-01-28 14:18:17,835 [121]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 14:18:17,837 [123]   INFO - login.py_run:30 - Starting Login Form...
2023-01-28 14:18:21,577 [3863]   INFO - login.py_quit:17 - Exit Pressed.
2023-01-28 14:18:23,525 [128]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 14:18:23,527 [130]   INFO - login.py_run:30 - Starting Login Form...
2023-01-28 14:18:27,846 [4448]   INFO - login.py_quit:17 - Exit Pressed.
2023-01-28 14:22:30,037 [122]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 14:22:30,039 [124]   INFO - login.py_run:30 - Starting Login Form...
2023-01-28 14:22:35,347 [5432]   INFO - login.py_quit:17 - Exit Pressed.
2023-01-28 14:24:28,743 [124]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 14:24:28,745 [126]   INFO - login.py_run:30 - Starting Login Form...
2023-01-28 14:24:37,911 [9292]   INFO - login.py_quit:17 - Exit Pressed.
2023-01-28 14:36:20,962 [238]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 14:36:20,965 [241]   INFO - login.py_run:54 - Starting Login Form...
2023-01-28 14:38:32,155 [208]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 14:38:32,157 [210]   INFO - login.py_run:54 - Starting Login Form...
2023-01-28 14:38:37,638 [5690]   INFO - login.py_doLogin:28 - Try Login, user=213213, pass=******
2023-01-28 14:38:37,639 [5691] WARNING - login.py_doLogin:39 - Login Failed, message: 用户名或密码错误
2023-01-28 14:38:47,145 [15198]   INFO - login.py_doLogin:28 - Try Login, user=213213, pass=******
2023-01-28 14:38:47,145 [15198] WARNING - login.py_doLogin:39 - Login Failed, message: 用户名或密码错误
2023-01-28 14:38:48,480 [16532]   INFO - login.py_doLogin:28 - Try Login, user=213213, pass=******
2023-01-28 14:38:48,480 [16532] WARNING - login.py_doLogin:39 - Login Failed, message: 用户名或密码错误
2023-01-28 14:38:49,975 [18027]   INFO - login.py_quit:20 - Exit Pressed.
2023-01-28 14:39:28,028 [209]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 14:39:28,030 [211]   INFO - login.py_run:54 - Starting Login Form...
2023-01-28 14:39:58,050 [215]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 14:39:58,052 [217]   INFO - login.py_run:54 - Starting Login Form...
2023-01-28 14:40:02,496 [4662]   INFO - login.py_doLogin:28 - Try Login, user=211, pass=******
2023-01-28 14:40:02,496 [4662] WARNING - login.py_doLogin:39 - Login Failed, message: 用户名或密码错误
2023-01-28 14:40:03,717 [5883]   INFO - login.py_quit:20 - Exit Pressed.
2023-01-28 14:40:14,939 [211]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 14:40:14,942 [214]   INFO - login.py_run:54 - Starting Login Form...
2023-01-28 14:40:19,106 [4378]   INFO - login.py_doLogin:28 - Try Login, user=1212, pass=******
2023-01-28 14:40:19,107 [4379] WARNING - login.py_doLogin:39 - Login Failed, message: 用户名或密码错误
2023-01-28 14:40:21,597 [6868]   INFO - login.py_quit:20 - Exit Pressed.
2023-01-28 14:40:25,764 [295]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 14:40:25,768 [299]   INFO - login.py_run:54 - Starting Login Form...
2023-01-28 14:40:30,473 [5005]   INFO - login.py_doLogin:28 - Try Login, user=我切尔无群二群无, pass=******
2023-01-28 14:40:30,474 [5006] WARNING - login.py_doLogin:39 - Login Failed, message: 用户名或密码错误
2023-01-28 14:40:32,645 [7176]   INFO - login.py_quit:20 - Exit Pressed.
2023-01-28 14:40:35,842 [257]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 14:40:35,845 [260]   INFO - login.py_run:54 - Starting Login Form...
2023-01-28 14:40:40,745 [5161]   INFO - login.py_doLogin:28 - Try Login, user=驱蚊器翁群翁, pass=******
2023-01-28 14:40:40,745 [5161] WARNING - login.py_doLogin:39 - Login Failed, message: 用户名或密码错误
2023-01-28 14:40:41,562 [5977]   INFO - login.py_doLogin:28 - Try Login, user=驱蚊器翁群翁, pass=******
2023-01-28 14:40:41,562 [5977] WARNING - login.py_doLogin:39 - Login Failed, message: 用户名或密码错误
2023-01-28 14:40:42,523 [6939]   INFO - login.py_doLogin:28 - Try Login, user=驱蚊器翁群翁, pass=******
2023-01-28 14:40:42,523 [6939] WARNING - login.py_doLogin:39 - Login Failed, message: 用户名或密码错误
2023-01-28 14:40:47,999 [12414]   INFO - login.py_doLogin:28 - Try Login, user=admin, pass=******
2023-01-28 14:40:47,999 [12414]   INFO - login.py_doLogin:32 - Logged in successfully, token=tok***
2023-01-28 14:40:52,674 [17090]   INFO - login.py_doLogin:28 - Try Login, user=admin, pass=******
2023-01-28 14:40:52,674 [17090]   INFO - login.py_doLogin:32 - Logged in successfully, token=tok***
2023-01-28 14:44:02,933 [202]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 14:44:02,937 [206]   INFO - login.py_run:54 - Starting Login Form...
2023-01-28 14:44:06,564 [3833]   INFO - login.py_quit:20 - Exit Pressed.
2023-01-28 14:49:22,489 [208]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 14:49:22,492 [211]   INFO - login.py_run:54 - Starting Login Form...
2023-01-28 14:49:38,251 [209]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 14:49:38,254 [212]   INFO - login.py_run:54 - Starting Login Form...
2023-01-28 14:49:42,866 [4824]   INFO - login.py_doLogin:28 - Try Login, user=21312, pass=******
2023-01-28 14:49:42,866 [4824] WARNING - login.py_doLogin:39 - Login Failed, message: 用户名或密码错误
2023-01-28 14:49:44,446 [6403]   INFO - login.py_quit:20 - Exit Pressed.
2023-01-28 14:49:54,353 [203]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 14:49:54,356 [206]   INFO - login.py_run:54 - Starting Login Form...
2023-01-28 14:49:58,240 [4090]   INFO - login.py_doLogin:28 - Try Login, user=213123, pass=******
2023-01-28 14:49:58,240 [4090] WARNING - login.py_doLogin:39 - Login Failed, message: 用户名或密码错误
2023-01-28 14:50:04,588 [210]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 14:50:04,590 [212]   INFO - login.py_run:54 - Starting Login Form...
2023-01-28 14:50:08,481 [4103]   INFO - login.py_doLogin:28 - Try Login, user=123123, pass=******
2023-01-28 14:50:08,481 [4103] WARNING - login.py_doLogin:39 - Login Failed, message: 用户名或密码错误
2023-01-28 14:50:10,217 [5839]   INFO - login.py_doLogin:28 - Try Login, user=123123, pass=******
2023-01-28 14:50:10,217 [5839] WARNING - login.py_doLogin:39 - Login Failed, message: 用户名或密码错误
2023-01-28 14:50:10,722 [6344]   INFO - login.py_doLogin:28 - Try Login, user=123123, pass=******
2023-01-28 14:50:10,723 [6344] WARNING - login.py_doLogin:39 - Login Failed, message: 用户名或密码错误
2023-01-28 14:50:15,140 [10762]   INFO - login.py_doLogin:28 - Try Login, user=123123, pass=******
2023-01-28 14:50:15,141 [10763] WARNING - login.py_doLogin:39 - Login Failed, message: 用户名或密码错误
2023-01-28 14:50:17,002 [12624]   INFO - login.py_quit:20 - Exit Pressed.
2023-01-28 15:01:34,806 [221]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 15:01:34,811 [226]   INFO - login.py_run:72 - Starting Login Form...
2023-01-28 15:01:45,482 [10897]   INFO - login.py_doLogin:28 - Try Login, user=admin, pass=******
2023-01-28 15:01:45,483 [10898] WARNING - login.py_doLogin:46 - Login Failed, message: 用户名或密码错误
2023-01-28 15:01:45,483 [10898]  ERROR - user.py_saveUser:32 - Save User Failed, [Errno 2] No such file or directory: './data/user.dat'
2023-01-28 15:01:49,310 [14725]   INFO - login.py_quit:20 - Exit Pressed.
2023-01-28 15:03:47,542 [201]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 15:03:47,545 [204]   INFO - login.py_run:72 - Starting Login Form...
2023-01-28 15:03:52,041 [4700]   INFO - login.py_doLogin:28 - Try Login, user=1231231, pass=******
2023-01-28 15:03:52,042 [4701] WARNING - login.py_doLogin:46 - Login Failed, message: 用户名或密码错误
2023-01-28 15:03:52,042 [4701]  ERROR - user.py_saveUser:30 - Save User Failed, name 'USER_PATH' is not defined
2023-01-28 15:03:53,131 [5789]   INFO - login.py_quit:20 - Exit Pressed.
2023-01-28 15:04:04,144 [207]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 15:04:04,146 [209]   INFO - login.py_run:72 - Starting Login Form...
2023-01-28 15:04:10,083 [6146]   INFO - login.py_doLogin:28 - Try Login, user=admin, pass=******
2023-01-28 15:04:10,083 [6146] WARNING - login.py_doLogin:46 - Login Failed, message: 用户名或密码错误
2023-01-28 15:04:11,518 [7581]   INFO - login.py_quit:20 - Exit Pressed.
2023-01-28 15:04:13,010 [253]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 15:04:13,013 [256]   INFO - login.py_run:72 - Starting Login Form...
2023-01-28 15:04:16,316 [3560]   INFO - login.py_quit:20 - Exit Pressed.
2023-01-28 15:05:17,537 [211]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 15:05:17,540 [214]   INFO - login.py_run:72 - Starting Login Form...
2023-01-28 15:05:21,786 [4460]   INFO - login.py_quit:20 - Exit Pressed.
2023-01-28 15:06:17,458 [225]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 15:06:17,461 [228]   INFO - login.py_run:72 - Starting Login Form...
2023-01-28 15:06:22,488 [5256]   INFO - login.py_quit:20 - Exit Pressed.
2023-01-28 15:06:50,663 [207]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 15:06:50,666 [210]   INFO - login.py_run:73 - Starting Login Form...
2023-01-28 15:06:57,185 [6728]   INFO - login.py_quit:20 - Exit Pressed.
2023-01-28 15:07:21,728 [206]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 15:07:21,730 [208]   INFO - login.py_run:73 - Starting Login Form...
2023-01-28 15:08:15,201 [210]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 15:08:15,203 [212]   INFO - login.py_run:74 - Starting Login Form...
2023-01-28 15:09:06,009 [149]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 15:09:06,011 [151]   INFO - login.py_run:74 - Starting Login Form...
2023-01-28 15:10:17,592 [145]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 15:10:17,594 [147]   INFO - login.py_run:74 - Starting Login Form...
2023-01-28 15:10:20,076 [2630]   INFO - login.py_getUser:55 - Load User From File
2023-01-28 15:10:20,077 [2631]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 15:10:56,452 [132]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 15:10:56,454 [134]   INFO - login.py_run:75 - Starting Login Form...
2023-01-28 15:10:58,813 [2493]   INFO - login.py_getUser:56 - Load User From File
2023-01-28 15:10:58,814 [2494]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 15:11:05,972 [9652]   INFO - login.py_doLogin:29 - Try Login, user=admin, pass=******
2023-01-28 15:11:05,972 [9652] WARNING - login.py_doLogin:48 - Login Failed, message: 用户名或密码错误
2023-01-28 15:11:05,973 [9653]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 15:11:06,989 [10669]   INFO - login.py_quit:21 - Exit Pressed.
2023-01-28 15:11:08,986 [129]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 15:11:08,988 [131]   INFO - login.py_run:75 - Starting Login Form...
2023-01-28 15:11:11,413 [2555]   INFO - login.py_getUser:56 - Load User From File
2023-01-28 15:11:11,417 [2559]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 15:11:17,704 [8847]   INFO - login.py_doLogin:29 - Try Login, user=admin, pass=******
2023-01-28 15:11:17,705 [8848]   INFO - login.py_doLogin:41 - Logged in successfully, token=tok***
2023-01-28 15:11:17,705 [8848]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 15:11:19,662 [10805]   INFO - login.py_quit:21 - Exit Pressed.
2023-01-28 15:11:20,955 [128]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 15:11:20,957 [130]   INFO - login.py_run:75 - Starting Login Form...
2023-01-28 15:11:23,431 [2604]   INFO - login.py_getUser:56 - Load User From File
2023-01-28 15:11:23,432 [2605]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 15:11:25,428 [4601]   INFO - login.py_doLogin:29 - Try Login, user=admin, pass=******
2023-01-28 15:11:25,428 [4601]   INFO - login.py_doLogin:41 - Logged in successfully, token=tok***
2023-01-28 15:11:25,429 [4602]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 15:11:26,329 [5502]   INFO - login.py_quit:21 - Exit Pressed.
2023-01-28 15:11:27,905 [125]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 15:11:27,907 [127]   INFO - login.py_run:75 - Starting Login Form...
2023-01-28 15:11:30,284 [2504]   INFO - login.py_getUser:56 - Load User From File
2023-01-28 15:11:30,285 [2505]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 15:11:32,027 [4247]   INFO - login.py_quit:21 - Exit Pressed.
2023-01-28 15:15:22,430 [131]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 15:15:22,432 [133]   INFO - login.py_run:75 - Starting Login Form...
2023-01-28 15:15:24,891 [2592]   INFO - login.py_getUser:56 - Load User From File
2023-01-28 15:15:24,895 [2596]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 15:15:26,179 [3879]   INFO - login.py_quit:21 - Exit Pressed.
2023-01-28 15:15:31,798 [128]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 15:15:31,800 [130]   INFO - login.py_run:75 - Starting Login Form...
2023-01-28 15:15:34,330 [2659]   INFO - login.py_getUser:56 - Load User From File
2023-01-28 15:15:34,331 [2660]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 15:15:34,441 [2770]   INFO - login.py_doLogin:29 - Try Login, user=admin, pass=******
2023-01-28 15:15:34,441 [2770]   INFO - login.py_doLogin:41 - Logged in successfully, token=tok***
2023-01-28 15:15:34,442 [2771]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 15:15:39,094 [7424]   INFO - login.py_quit:21 - Exit Pressed.
2023-01-28 15:24:42,211 [128]   INFO - __main__.py_<module>:12 - Program Loading.
2023-01-28 15:24:42,214 [131]   INFO - login.py_run:75 - Starting Login Form...
2023-01-28 15:24:44,642 [2559]   INFO - login.py_getUser:56 - Load User From File
2023-01-28 15:24:44,647 [2564]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 15:24:44,749 [2667]   INFO - login.py_doLogin:29 - Try Login, user=admin, pass=******
2023-01-28 15:24:44,749 [2667]   INFO - login.py_doLogin:41 - Logged in successfully, token=tok***
2023-01-28 15:24:44,749 [2667]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 15:24:46,193 [4110]   INFO - login.py_quit:21 - Exit Pressed.
2023-01-28 15:39:21,813 [127]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 15:39:38,598 [125]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 15:39:38,600 [127]   INFO - monitor.py_run:34 - Starting Monitor Form...
2023-01-28 15:39:56,583 [122]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 15:39:56,585 [124]   INFO - monitor.py_run:34 - Starting Monitor Form...
2023-01-28 15:43:57,638 [130]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 15:43:57,639 [131]   INFO - monitor.py_run:34 - Starting Monitor Form...
2023-01-28 15:44:15,358 [127]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 15:44:15,359 [128]   INFO - monitor.py_run:34 - Starting Monitor Form...
2023-01-28 15:48:41,648 [256]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 15:48:41,650 [258]   INFO - monitor.py_run:35 - Starting Monitor Form...
2023-01-28 15:49:09,000 [252]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 15:49:09,002 [254]   INFO - monitor.py_run:35 - Starting Monitor Form...
2023-01-28 15:51:12,230 [386]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 15:51:12,232 [388]   INFO - monitor.py_run:39 - Starting Monitor Form...
2023-01-28 15:51:28,396 [306]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 15:51:28,398 [308]   INFO - monitor.py_run:41 - Starting Monitor Form...
2023-01-28 16:34:05,478 [294]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 16:34:05,481 [297]   INFO - monitor.py_run:34 - Starting Monitor Form...
2023-01-28 16:34:46,887 [297]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 16:34:46,889 [299]   INFO - monitor.py_run:34 - Starting Monitor Form...
2023-01-28 16:35:17,381 [286]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 16:35:17,392 [297]   INFO - monitor.py_run:34 - Starting Monitor Form...
2023-01-28 16:35:19,863 [2768]   INFO - monitor.py_getCameraList:21 - Get Camera List...
2023-01-28 16:35:19,886 [2791]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 16:37:20,762 [297]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 16:37:20,764 [299]   INFO - monitor.py_run:42 - Starting Monitor Form...
2023-01-28 16:37:23,264 [2800]   INFO - monitor.py_getCameraList:21 - Get Camera List...
2023-01-28 16:37:23,288 [2824]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 16:37:41,525 [293]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 16:37:41,527 [295]   INFO - monitor.py_run:42 - Starting Monitor Form...
2023-01-28 16:37:44,031 [2799]   INFO - monitor.py_getCameraList:21 - Get Camera List...
2023-01-28 16:37:44,054 [2821]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 16:38:02,419 [283]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 16:38:02,422 [286]   INFO - monitor.py_run:42 - Starting Monitor Form...
2023-01-28 16:38:04,876 [2740]   INFO - monitor.py_getCameraList:21 - Get Camera List...
2023-01-28 16:38:04,898 [2762]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 16:39:43,048 [274]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 16:39:43,050 [276]   INFO - monitor.py_run:42 - Starting Monitor Form...
2023-01-28 16:39:45,467 [2693]   INFO - monitor.py_getCameraList:21 - Get Camera List...
2023-01-28 16:39:45,492 [2718]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 16:45:58,539 [291]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 16:45:58,542 [294]   INFO - monitor.py_run:48 - Starting Monitor Form...
2023-01-28 16:46:01,093 [2845]   INFO - monitor.py_getCameraList:21 - Get Camera List...
2023-01-28 16:46:01,116 [2868]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 16:47:10,467 [318]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 16:47:10,470 [321]   INFO - monitor.py_run:48 - Starting Monitor Form...
2023-01-28 16:47:13,001 [2852]   INFO - monitor.py_getCameraList:21 - Get Camera List...
2023-01-28 16:47:13,025 [2876]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 16:47:29,807 [296]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 16:47:29,809 [298]   INFO - monitor.py_run:48 - Starting Monitor Form...
2023-01-28 16:47:32,252 [2742]   INFO - monitor.py_getCameraList:21 - Get Camera List...
2023-01-28 16:47:32,273 [2763]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 16:47:36,719 [7209]   INFO - monitor.py___init__:44 - Start Video Capture on Camera-0
2023-01-28 16:47:40,865 [11354]   INFO - monitor.py_getFreePort:62 - Got Free Port59786
2023-01-28 16:47:40,866 [11355]   INFO - monitor.py_getFlaskThread:97 - Get Flask Instance on 127.0.0.1:59786
2023-01-28 16:47:40,866 [11355]  ERROR - monitor.py_getCameraURL:37 - Open Camera Failed:can only concatenate str (not "int") to str
2023-01-28 16:48:11,760 [288]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 16:48:11,762 [290]   INFO - monitor.py_run:48 - Starting Monitor Form...
2023-01-28 16:48:14,189 [2716]   INFO - monitor.py_getCameraList:21 - Get Camera List...
2023-01-28 16:48:14,211 [2739]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 16:48:19,043 [7571]   INFO - monitor.py___init__:44 - Start Video Capture on Camera-0
2023-01-28 16:48:23,084 [11611]   INFO - monitor.py_getFreePort:62 - Got Free Port59926
2023-01-28 16:48:23,085 [11612]   INFO - monitor.py_getFlaskThread:97 - Get Flask Instance on 127.0.0.1:59926
2023-01-28 16:48:23,085 [11612]  ERROR - monitor.py_getCameraURL:37 - Open Camera Failed:can only concatenate str (not "int") to str
2023-01-28 16:52:48,125 [293]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 16:52:48,127 [295]   INFO - monitor.py_run:48 - Starting Monitor Form...
2023-01-28 16:52:50,601 [2770]   INFO - monitor.py_getCameraList:21 - Get Camera List...
2023-01-28 16:52:50,626 [2795]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 16:52:56,731 [8899]   INFO - monitor.py___init__:44 - Start Video Capture on Camera-0
2023-01-28 16:53:00,908 [13076]   INFO - monitor.py_getFreePort:62 - Got Free Port60092
2023-01-28 16:53:00,908 [13077]   INFO - monitor.py_getFlaskThread:97 - Get Flask Instance on 127.0.0.1:60092
2023-01-28 16:55:01,208 [280]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 16:55:01,210 [282]   INFO - monitor.py_run:48 - Starting Monitor Form...
2023-01-28 16:55:03,721 [2794]   INFO - monitor.py_getCameraList:21 - Get Camera List...
2023-01-28 16:55:03,742 [2815]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 16:55:05,765 [4838]   INFO - monitor.py___init__:44 - Start Video Capture on Camera-0
2023-01-28 16:55:09,775 [8848]   INFO - monitor.py_getFreePort:62 - Got Free Port60178
2023-01-28 16:55:09,776 [8849]   INFO - monitor.py_getFlaskThread:97 - Get Flask Instance on 127.0.0.1:60178
2023-01-28 16:55:17,879 [16952]   INFO - monitor.py___init__:44 - Start Video Capture on Camera-0
2023-01-28 16:55:21,252 [20325]   INFO - monitor.py_getFlaskThread:86 - Thread Exists, Killing...
2023-01-28 16:55:21,252 [20325]  ERROR - monitor.py_getCameraURL:37 - Open Camera Failed:can only concatenate str (not "int") to str
2023-01-28 16:55:43,851 [309]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 16:55:43,853 [311]   INFO - monitor.py_run:48 - Starting Monitor Form...
2023-01-28 16:55:46,398 [2855]   INFO - monitor.py_getCameraList:21 - Get Camera List...
2023-01-28 16:55:46,424 [2881]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 16:57:35,555 [317]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 16:57:35,557 [319]   INFO - monitor.py_run:55 - Starting Monitor Form...
2023-01-28 16:57:38,155 [2917]   INFO - monitor.py_getCameraList:21 - Get Camera List...
2023-01-28 16:57:38,178 [2940]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 16:57:41,051 [5814]   INFO - monitor.py___init__:44 - Start Video Capture on Camera-0
2023-01-28 16:57:45,057 [9819]   INFO - monitor.py_getFreePort:62 - Got Free Port60462
2023-01-28 16:57:45,058 [9820]   INFO - monitor.py_getFlaskThread:97 - Get Flask Instance on 127.0.0.1:60462
2023-01-28 16:58:47,507 [320]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 16:58:47,509 [322]   INFO - monitor.py_run:55 - Starting Monitor Form...
2023-01-28 16:58:50,097 [2910]   INFO - monitor.py_getCameraList:21 - Get Camera List...
2023-01-28 16:58:50,119 [2931]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 16:58:54,227 [7040]   INFO - monitor.py___init__:44 - Start Video Capture on Camera-0
2023-01-28 16:58:57,078 [9890]   INFO - monitor.py___init__:44 - Start Video Capture on Camera-0
2023-01-28 16:58:58,593 [11406]   INFO - monitor.py_getFreePort:62 - Got Free Port60635
2023-01-28 16:58:58,595 [11408]   INFO - monitor.py_getFlaskThread:97 - Get Flask Instance on 127.0.0.1:60635
2023-01-28 16:59:00,875 [13688]   INFO - monitor.py_getFlaskThread:86 - Thread Exists, Killing...
2023-01-28 16:59:00,876 [13689]  ERROR - monitor.py_getCameraURL:40 - Open Camera Failed:can only concatenate str (not "int") to str
2023-01-28 16:59:00,876 [13689]   INFO - monitor.py___del__:48 - Stop Video Capture
2023-01-28 16:59:25,206 [286]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 16:59:25,208 [288]   INFO - monitor.py_run:55 - Starting Monitor Form...
2023-01-28 16:59:27,686 [2765]   INFO - monitor.py_getCameraList:21 - Get Camera List...
2023-01-28 16:59:27,709 [2788]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 16:59:29,711 [4790]   INFO - monitor.py___init__:44 - Start Video Capture on Camera-0
2023-01-28 16:59:33,653 [8733]   INFO - monitor.py_getFreePort:62 - Got Free Port60690
2023-01-28 16:59:33,654 [8734]   INFO - monitor.py_getFlaskThread:97 - Get Flask Instance on 127.0.0.1:60690
2023-01-28 17:03:30,916 [292]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 17:03:30,918 [294]   INFO - monitor.py_run:60 - Starting Monitor Form...
2023-01-28 17:03:33,528 [2904]   INFO - monitor.py_getCameraList:21 - Get Camera List...
2023-01-28 17:03:33,550 [2926]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 17:03:37,876 [7252]   INFO - monitor.py_getCameraURL:34 - Get Camera URL with Camera-0
2023-01-28 17:03:37,876 [7252]   INFO - monitor.py___init__:44 - Start Video Capture on Camera-0
2023-01-28 17:03:41,952 [11327]   INFO - monitor.py_getFreePort:62 - Got Free Port60890
2023-01-28 17:03:41,953 [11328]   INFO - monitor.py_getFlaskThread:97 - Get Flask Instance on 127.0.0.1:60890
2023-01-28 17:10:30,698 [288]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 17:10:30,701 [291]   INFO - monitor.py_run:60 - Starting Monitor Form...
2023-01-28 17:10:33,201 [2790]   INFO - monitor.py_getCameraList:21 - Get Camera List...
2023-01-28 17:10:33,223 [2812]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 17:11:32,024 [287]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 17:11:32,027 [290]   INFO - monitor.py_run:60 - Starting Monitor Form...
2023-01-28 17:11:34,486 [2749]   INFO - monitor.py_getCameraList:21 - Get Camera List...
2023-01-28 17:11:34,514 [2777]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 17:11:36,426 [4689]   INFO - monitor.py_getCameraURL:34 - Get Camera URL with Camera-0
2023-01-28 17:11:36,427 [4690]   INFO - monitor.py___init__:44 - Start Video Capture on Camera-0
2023-01-28 17:11:40,436 [8699]   INFO - monitor.py_getFreePort:62 - Got Free Port61572
2023-01-28 17:11:40,437 [8700]   INFO - monitor.py_getFlaskThread:97 - Get Flask Instance on 127.0.0.1:61572
2023-01-28 17:12:42,419 [288]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 17:12:42,421 [290]   INFO - monitor.py_run:60 - Starting Monitor Form...
2023-01-28 17:12:44,858 [2727]   INFO - monitor.py_getCameraList:21 - Get Camera List...
2023-01-28 17:12:44,879 [2748]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 17:12:49,160 [7029]   INFO - monitor.py_getCameraURL:34 - Get Camera URL with Camera-0
2023-01-28 17:12:49,161 [7030]   INFO - monitor.py___init__:44 - Start Video Capture on Camera-0
2023-01-28 17:12:53,141 [11010]   INFO - monitor.py_getFreePort:62 - Got Free Port61675
2023-01-28 17:12:53,142 [11011]   INFO - monitor.py_getFlaskThread:97 - Get Flask Instance on 127.0.0.1:61675
2023-01-28 17:13:14,649 [289]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 17:13:14,651 [291]   INFO - monitor.py_run:60 - Starting Monitor Form...
2023-01-28 17:13:17,168 [2808]   INFO - monitor.py_getCameraList:21 - Get Camera List...
2023-01-28 17:13:17,191 [2831]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 17:13:18,972 [4612]   INFO - monitor.py_getCameraURL:34 - Get Camera URL with Camera-0
2023-01-28 17:13:18,972 [4612]   INFO - monitor.py___init__:44 - Start Video Capture on Camera-0
2023-01-28 17:13:22,849 [8489]   INFO - monitor.py_getFreePort:62 - Got Free Port61708
2023-01-28 17:13:22,850 [8490]   INFO - monitor.py_getFlaskThread:97 - Get Flask Instance on 127.0.0.1:61708
2023-01-28 17:13:35,642 [281]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 17:13:35,643 [283]   INFO - monitor.py_run:60 - Starting Monitor Form...
2023-01-28 17:13:38,157 [2797]   INFO - monitor.py_getCameraList:21 - Get Camera List...
2023-01-28 17:13:38,178 [2818]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 17:13:40,116 [4756]   INFO - monitor.py_getCameraURL:34 - Get Camera URL with Camera-0
2023-01-28 17:13:40,117 [4757]   INFO - monitor.py___init__:44 - Start Video Capture on Camera-0
2023-01-28 17:13:44,109 [8748]   INFO - monitor.py_getFreePort:62 - Got Free Port61746
2023-01-28 17:13:44,110 [8749]   INFO - monitor.py_getFlaskThread:97 - Get Flask Instance on 127.0.0.1:61746
2023-01-28 17:15:27,571 [287]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 17:15:27,573 [289]   INFO - monitor.py_run:64 - Starting Monitor Form...
2023-01-28 17:15:29,997 [2713]   INFO - monitor.py_getCameraList:22 - Get Camera List...
2023-01-28 17:15:30,019 [2735]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 17:15:51,539 [296]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 17:15:51,541 [298]   INFO - monitor.py_run:67 - Starting Monitor Form...
2023-01-28 17:15:54,064 [2821]   INFO - monitor.py_getCameraList:22 - Get Camera List...
2023-01-28 17:15:54,085 [2842]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 17:16:54,478 [303]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 17:16:54,480 [305]   INFO - login.py_run:77 - Starting Login Form...
2023-01-28 17:16:56,696 [2521]   INFO - login.py_getUser:58 - Load User From File
2023-01-28 17:16:56,697 [2522]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 17:16:56,929 [2755]   INFO - login.py_doLogin:29 - Try Login, user=admin, pass=******
2023-01-28 17:16:56,929 [2755]   INFO - login.py_doLogin:41 - Logged in successfully, token=tok***
2023-01-28 17:16:56,930 [2756]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 17:16:56,931 [2757] WARNING - login.py_doLogin:50 - Login Failed, message: module 'webview.http' has no attribute 'running'
2023-01-28 17:16:56,931 [2757]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 17:16:59,766 [5592]   INFO - login.py_doLogin:29 - Try Login, user=admin, pass=******
2023-01-28 17:16:59,766 [5592]   INFO - login.py_doLogin:41 - Logged in successfully, token=tok***
2023-01-28 17:16:59,767 [5593]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 17:16:59,767 [5593] WARNING - login.py_doLogin:50 - Login Failed, message: module 'webview.http' has no attribute 'running'
2023-01-28 17:16:59,768 [5594]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 17:17:39,579 [293]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 17:17:39,581 [295]   INFO - login.py_run:77 - Starting Login Form...
2023-01-28 17:17:41,989 [2703]   INFO - login.py_getUser:58 - Load User From File
2023-01-28 17:17:41,990 [2704]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 17:17:42,092 [2806]   INFO - login.py_doLogin:29 - Try Login, user=admin, pass=******
2023-01-28 17:17:42,092 [2806]   INFO - login.py_doLogin:41 - Logged in successfully, token=tok***
2023-01-28 17:17:42,093 [2807]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 17:17:42,093 [2807] WARNING - login.py_doLogin:50 - Login Failed, message: module 'webview.http' has no attribute 'running'
2023-01-28 17:17:42,094 [2808]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 17:17:43,956 [4670]   INFO - login.py_doLogin:29 - Try Login, user=admin, pass=******
2023-01-28 17:17:43,956 [4670]   INFO - login.py_doLogin:41 - Logged in successfully, token=tok***
2023-01-28 17:17:43,957 [4671]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 17:17:43,957 [4671] WARNING - login.py_doLogin:50 - Login Failed, message: module 'webview.http' has no attribute 'running'
2023-01-28 17:17:43,957 [4671]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 17:18:20,483 [294]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 17:18:20,485 [296]   INFO - login.py_run:78 - Starting Login Form...
2023-01-28 17:18:22,884 [2695]   INFO - login.py_getUser:59 - Load User From File
2023-01-28 17:18:22,885 [2696]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 17:18:22,992 [2803]   INFO - login.py_doLogin:29 - Try Login, user=admin, pass=******
2023-01-28 17:18:22,993 [2804]   INFO - login.py_doLogin:41 - Logged in successfully, token=tok***
2023-01-28 17:18:22,994 [2805]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 17:18:23,065 [2876] WARNING - login.py_doLogin:51 - Login Failed, message: module 'webview.http' has no attribute 'running'
2023-01-28 17:18:23,065 [2876]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:22:01,031 [440]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:22:01,032 [441]   INFO - login.py_run:77 - Starting Login Form...
2023-01-28 22:22:04,621 [4031]   INFO - login.py_getUser:58 - Load User From File
2023-01-28 22:22:04,621 [4031]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:22:04,713 [4122]   INFO - login.py_doLogin:29 - Try Login, user=admin, pass=******
2023-01-28 22:22:04,713 [4122]   INFO - login.py_doLogin:41 - Logged in successfully, token=tok***
2023-01-28 22:22:04,714 [4123]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:22:04,794 [4203] WARNING - login.py_doLogin:50 - Login Failed, message: module 'webview.http' has no attribute 'running'
2023-01-28 22:22:04,794 [4203]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:22:38,752 [291]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:22:38,754 [293]   INFO - login.py_run:77 - Starting Login Form...
2023-01-28 22:22:41,194 [2733]   INFO - login.py_getUser:58 - Load User From File
2023-01-28 22:22:41,195 [2734]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:22:41,297 [2836]   INFO - login.py_doLogin:29 - Try Login, user=admin, pass=******
2023-01-28 22:22:41,297 [2836]   INFO - login.py_doLogin:41 - Logged in successfully, token=tok***
2023-01-28 22:22:41,298 [2837]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:22:41,299 [2838] WARNING - login.py_doLogin:50 - Login Failed, message: module 'webview.http' has no attribute 'running'
2023-01-28 22:22:41,300 [2839]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:22:56,609 [297]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:22:56,611 [299]   INFO - login.py_run:78 - Starting Login Form...
2023-01-28 22:22:59,058 [2746]   INFO - login.py_getUser:59 - Load User From File
2023-01-28 22:22:59,058 [2746]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:22:59,167 [2855]   INFO - login.py_doLogin:29 - Try Login, user=admin, pass=******
2023-01-28 22:22:59,167 [2855]   INFO - login.py_doLogin:41 - Logged in successfully, token=tok***
2023-01-28 22:22:59,168 [2856]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:22:59,171 [2859] WARNING - login.py_doLogin:51 - Login Failed, message: module 'webview.http' has no attribute 'running'
2023-01-28 22:22:59,172 [2860]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:23:46,652 [307]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:23:46,654 [309]   INFO - login.py_run:78 - Starting Login Form...
2023-01-28 22:23:49,188 [2843]   INFO - login.py_getUser:59 - Load User From File
2023-01-28 22:23:49,189 [2844]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:23:49,291 [2946]   INFO - login.py_doLogin:29 - Try Login, user=admin, pass=******
2023-01-28 22:23:49,291 [2946]   INFO - login.py_doLogin:41 - Logged in successfully, token=tok***
2023-01-28 22:23:49,292 [2947]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:23:49,295 [2950] WARNING - login.py_doLogin:51 - Login Failed, message: module 'webview.http' has no attribute 'running'
2023-01-28 22:23:49,295 [2950]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:24:31,911 [330]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:24:31,912 [331]   INFO - login.py_run:78 - Starting Login Form...
2023-01-28 22:24:34,030 [2448]   INFO - login.py_getUser:59 - Load User From File
2023-01-28 22:24:34,030 [2448]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:24:34,138 [2556]   INFO - login.py_doLogin:29 - Try Login, user=admin, pass=******
2023-01-28 22:24:34,138 [2556]   INFO - login.py_doLogin:41 - Logged in successfully, token=tok***
2023-01-28 22:24:34,139 [2557]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:24:34,162 [2580] WARNING - login.py_doLogin:51 - Login Failed, message: module 'webview.http' has no attribute 'running'
2023-01-28 22:24:34,163 [2581]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:25:42,142 [287]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:25:42,145 [290]   INFO - login.py_run:78 - Starting Login Form...
2023-01-28 22:25:51,039 [291]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:25:51,041 [292]   INFO - login.py_run:78 - Starting Login Form...
2023-01-28 22:25:53,048 [2299]   INFO - login.py_getUser:59 - Load User From File
2023-01-28 22:25:53,048 [2299]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:25:53,153 [2405]   INFO - login.py_doLogin:29 - Try Login, user=admin, pass=******
2023-01-28 22:25:53,154 [2406]   INFO - login.py_doLogin:41 - Logged in successfully, token=tok***
2023-01-28 22:25:53,155 [2407]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:25:53,174 [2426] WARNING - login.py_doLogin:51 - Login Failed, message: module 'webview.http' has no attribute 'running'
2023-01-28 22:25:53,175 [2427]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:26:29,988 [291]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:26:29,990 [293]   INFO - login.py_run:78 - Starting Login Form...
2023-01-28 22:26:32,130 [2433]   INFO - login.py_getUser:59 - Load User From File
2023-01-28 22:26:32,131 [2434]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:26:32,233 [2536]   INFO - login.py_doLogin:29 - Try Login, user=admin, pass=******
2023-01-28 22:26:32,233 [2536]   INFO - login.py_doLogin:41 - Logged in successfully, token=tok***
2023-01-28 22:26:32,234 [2537]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:26:32,249 [2552] WARNING - login.py_doLogin:51 - Login Failed, message: module 'webview.http' has no attribute 'running'
2023-01-28 22:26:32,250 [2553]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:27:08,914 [305]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:27:27,157 [297]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:27:40,907 [282]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:27:52,296 [285]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:29:34,190 [293]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:29:56,644 [295]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:31:21,334 [289]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:31:23,504 [2459]   INFO - login.py_getUser:59 - Load User From File
2023-01-28 22:31:23,504 [2459]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:31:23,606 [2562]   INFO - login.py_doLogin:29 - Try Login, user=admin, pass=******
2023-01-28 22:31:23,607 [2563]   INFO - login.py_doLogin:41 - Logged in successfully, token=tok***
2023-01-28 22:31:23,608 [2564]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:31:23,608 [2564] WARNING - login.py_doLogin:51 - Login Failed, message: 'NoneType' object has no attribute 'window'
2023-01-28 22:31:23,609 [2565]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:31:28,418 [7373]   INFO - login.py_quit:21 - Exit Pressed.
2023-01-28 22:31:29,063 [8018]   INFO - login.py_quit:21 - Exit Pressed.
2023-01-28 22:31:29,474 [8430]   INFO - login.py_quit:21 - Exit Pressed.
2023-01-28 22:31:32,003 [10959]   INFO - login.py_quit:21 - Exit Pressed.
2023-01-28 22:31:32,252 [11208]   INFO - login.py_quit:21 - Exit Pressed.
2023-01-28 22:31:32,677 [11632]   INFO - login.py_quit:21 - Exit Pressed.
2023-01-28 22:31:33,840 [12796]   INFO - login.py_quit:21 - Exit Pressed.
2023-01-28 22:31:34,494 [13449]   INFO - login.py_quit:21 - Exit Pressed.
2023-01-28 22:31:35,258 [14213]   INFO - login.py_quit:21 - Exit Pressed.
2023-01-28 22:31:49,708 [296]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:31:51,786 [2374]   INFO - login.py_getUser:59 - Load User From File
2023-01-28 22:31:51,787 [2375]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:31:51,882 [2470]   INFO - login.py_doLogin:29 - Try Login, user=admin, pass=******
2023-01-28 22:31:51,882 [2470]   INFO - login.py_doLogin:41 - Logged in successfully, token=tok***
2023-01-28 22:31:51,883 [2471]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:31:51,883 [2471] WARNING - login.py_doLogin:51 - Login Failed, message: 'NoneType' object has no attribute 'window'
2023-01-28 22:31:51,884 [2472]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:31:52,502 [3090]   INFO - login.py_quit:21 - Exit Pressed.
2023-01-28 22:31:53,760 [4348]   INFO - login.py_quit:21 - Exit Pressed.
2023-01-28 22:32:35,391 [319]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:32:37,441 [2369]   INFO - login.py_getUser:56 - Load User From File
2023-01-28 22:32:37,441 [2369]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:32:37,539 [2467]   INFO - login.py_doLogin:26 - Try Login, user=admin, pass=******
2023-01-28 22:32:37,540 [2468]   INFO - login.py_doLogin:38 - Logged in successfully, token=tok***
2023-01-28 22:32:37,541 [2469]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:32:37,541 [2469] WARNING - login.py_doLogin:48 - Login Failed, message: 'JsAPI' object has no attribute 'parent'
2023-01-28 22:32:37,541 [2469]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:32:38,590 [3518]   INFO - login.py_quit:18 - Exit Pressed.
2023-01-28 22:36:22,733 [294]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:36:28,890 [295]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:36:34,521 [298]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:36:36,662 [2438]   INFO - login.py_getUser:57 - Load User From File
2023-01-28 22:36:36,663 [2439]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:36:36,688 [2465]   INFO - monitor.py_getCameraList:13 - Get Camera List...
2023-01-28 22:36:36,709 [2486]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 22:36:36,771 [2548]   INFO - login.py_doLogin:27 - Try Login, user=admin, pass=******
2023-01-28 22:36:36,772 [2549]   INFO - login.py_doLogin:39 - Logged in successfully, token=tok***
2023-01-28 22:36:36,772 [2549]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:36:36,773 [2550] WARNING - login.py_doLogin:49 - Login Failed, message: 'JsAPI' object has no attribute 'parent'
2023-01-28 22:36:36,773 [2550]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:37:02,633 [298]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:37:33,458 [368]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:37:35,550 [2460]   INFO - login.py_getUser:55 - Load User From File
2023-01-28 22:37:35,550 [2460]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:37:35,665 [2576]   INFO - login.py_doLogin:27 - Try Login, user=admin, pass=******
2023-01-28 22:37:35,665 [2576]   INFO - login.py_doLogin:39 - Logged in successfully, token=tok***
2023-01-28 22:37:35,666 [2577]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:37:35,666 [2577] WARNING - login.py_doLogin:47 - Login Failed, message: 'NoneType' object has no attribute 'show'
2023-01-28 22:37:35,667 [2578]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:37:46,424 [310]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:37:48,671 [2556]   INFO - login.py_getUser:55 - Load User From File
2023-01-28 22:37:48,671 [2556]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:37:48,698 [2583]   INFO - monitor.py_getCameraList:13 - Get Camera List...
2023-01-28 22:37:48,718 [2604]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 22:37:48,777 [2662]   INFO - login.py_doLogin:27 - Try Login, user=admin, pass=******
2023-01-28 22:37:48,778 [2663]   INFO - login.py_doLogin:39 - Logged in successfully, token=tok***
2023-01-28 22:37:48,778 [2663]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:38:04,251 [297]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:38:27,575 [296]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:38:29,654 [2375]   INFO - login.py_getUser:56 - Load User From File
2023-01-28 22:38:29,654 [2375]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:38:29,714 [2435]   INFO - monitor.py_getCameraList:13 - Get Camera List...
2023-01-28 22:38:29,736 [2457]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 22:38:29,765 [2486]   INFO - login.py_doLogin:27 - Try Login, user=admin, pass=******
2023-01-28 22:38:29,765 [2486]   INFO - login.py_doLogin:39 - Logged in successfully, token=tok***
2023-01-28 22:38:29,766 [2487]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:38:33,239 [5960]   INFO - monitor.py_getCameraURL:26 - Get Camera URL with Camera-0
2023-01-28 22:38:33,239 [5960]   INFO - monitor.py___init__:44 - Start Video Capture on Camera-0
2023-01-28 22:38:37,734 [10455]   INFO - monitor.py_getFreePort:62 - Got Free Port58399
2023-01-28 22:38:37,735 [10456]   INFO - monitor.py_getFlaskThread:97 - Get Flask Instance on 127.0.0.1:58399
2023-01-28 22:38:46,730 [19451]   INFO - monitor.py_stopCapture:40 - Stop Monitoring...
2023-01-28 22:39:38,754 [303]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:39:40,921 [2471]   INFO - login.py_getUser:56 - Load User From File
2023-01-28 22:39:40,922 [2472]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:39:40,944 [2494]   INFO - monitor.py_getCameraList:13 - Get Camera List...
2023-01-28 22:39:40,964 [2514]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 22:39:41,032 [2581]   INFO - login.py_doLogin:27 - Try Login, user=admin, pass=******
2023-01-28 22:39:41,032 [2581]   INFO - login.py_doLogin:39 - Logged in successfully, token=tok***
2023-01-28 22:39:41,033 [2582]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:39:43,474 [5023]   INFO - monitor.py_getCameraURL:26 - Get Camera URL with Camera-0
2023-01-28 22:39:43,475 [5024]   INFO - monitor.py___init__:44 - Start Video Capture on Camera-0
2023-01-28 22:39:47,464 [9013]   INFO - monitor.py_getFreePort:62 - Got Free Port58438
2023-01-28 22:39:47,465 [9014]   INFO - monitor.py_getFlaskThread:97 - Get Flask Instance on 127.0.0.1:58438
2023-01-28 22:39:49,145 [10694]   INFO - monitor.py_stopCapture:40 - Stop Monitoring...
2023-01-28 22:39:49,145 [10694]   INFO - monitor.py_stopMonitoring:110 - <Thread(Thread-10, started daemon 2272)>
2023-01-28 22:40:37,271 [338]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:40:39,496 [2562]   INFO - login.py_getUser:56 - Load User From File
2023-01-28 22:40:39,497 [2563]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:40:39,519 [2585]   INFO - monitor.py_getCameraList:13 - Get Camera List...
2023-01-28 22:40:39,539 [2606]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 22:40:39,602 [2669]   INFO - login.py_doLogin:27 - Try Login, user=admin, pass=******
2023-01-28 22:40:39,602 [2669]   INFO - login.py_doLogin:39 - Logged in successfully, token=tok***
2023-01-28 22:40:39,603 [2670]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:40:41,467 [4534]   INFO - monitor.py_stopCapture:40 - Stop Monitoring...
2023-01-28 22:40:41,467 [4534]   INFO - monitor.py_stopMonitoring:110 - None
2023-01-28 22:41:23,294 [311]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:41:25,331 [2348]   INFO - login.py_getUser:56 - Load User From File
2023-01-28 22:41:25,332 [2349]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:41:25,433 [2450]   INFO - login.py_doLogin:27 - Try Login, user=admin, pass=******
2023-01-28 22:41:25,434 [2451]   INFO - login.py_doLogin:39 - Logged in successfully, token=tok***
2023-01-28 22:41:25,434 [2451]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:41:25,452 [2470] WARNING - login.py_doLogin:48 - Login Failed, message: 'NoneType' object has no attribute 'show'
2023-01-28 22:41:25,453 [2471]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:41:39,522 [311]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:41:41,593 [2381]   INFO - login.py_getUser:56 - Load User From File
2023-01-28 22:41:41,594 [2382]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:41:41,665 [2453]   INFO - monitor.py_getCameraList:13 - Get Camera List...
2023-01-28 22:41:41,685 [2474]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 22:41:41,704 [2493]   INFO - login.py_doLogin:27 - Try Login, user=admin, pass=******
2023-01-28 22:41:41,705 [2494]   INFO - login.py_doLogin:39 - Logged in successfully, token=tok***
2023-01-28 22:41:41,705 [2494]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:41:58,629 [312]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:42:00,685 [2369]   INFO - login.py_getUser:56 - Load User From File
2023-01-28 22:42:00,686 [2370]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:42:00,762 [2445]   INFO - monitor.py_getCameraList:13 - Get Camera List...
2023-01-28 22:42:00,782 [2465]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 22:42:00,785 [2469]   INFO - login.py_doLogin:27 - Try Login, user=admin, pass=******
2023-01-28 22:42:00,785 [2469]   INFO - login.py_doLogin:39 - Logged in successfully, token=tok***
2023-01-28 22:42:00,786 [2470]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:42:20,805 [22489] WARNING - login.py_doLogin:48 - Login Failed, message: Main window failed to start
2023-01-28 22:42:20,805 [22489]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:42:23,245 [313]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:42:25,388 [2456]   INFO - monitor.py_getCameraList:13 - Get Camera List...
2023-01-28 22:42:25,410 [2477]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 22:42:34,198 [286]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:42:36,337 [2424]   INFO - login.py_getUser:56 - Load User From File
2023-01-28 22:42:36,338 [2425]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:42:36,415 [2502]   INFO - monitor.py_getCameraList:13 - Get Camera List...
2023-01-28 22:42:36,437 [2524]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 22:42:36,444 [2531]   INFO - login.py_doLogin:27 - Try Login, user=admin, pass=******
2023-01-28 22:42:36,444 [2531]   INFO - login.py_doLogin:39 - Logged in successfully, token=tok***
2023-01-28 22:42:36,445 [2532]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:42:56,474 [22561] WARNING - login.py_doLogin:48 - Login Failed, message: Main window failed to start
2023-01-28 22:42:56,475 [22562]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:43:48,034 [293]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:43:50,107 [2366]   INFO - login.py_getUser:56 - Load User From File
2023-01-28 22:43:50,107 [2366]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:43:50,189 [2447]   INFO - monitor.py_getCameraList:13 - Get Camera List...
2023-01-28 22:43:50,202 [2460]   INFO - login.py_doLogin:27 - Try Login, user=admin, pass=******
2023-01-28 22:43:50,202 [2460]   INFO - login.py_doLogin:39 - Logged in successfully, token=tok***
2023-01-28 22:43:50,203 [2461]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:43:50,210 [2468]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 22:44:10,237 [22495] WARNING - login.py_doLogin:48 - Login Failed, message: Main window failed to start
2023-01-28 22:44:10,238 [22496]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:44:18,188 [302]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:44:20,269 [2383]   INFO - login.py_getUser:56 - Load User From File
2023-01-28 22:44:20,270 [2384]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:44:20,294 [2408]   INFO - monitor.py_getCameraList:13 - Get Camera List...
2023-01-28 22:44:20,314 [2428]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 22:44:20,368 [2482]   INFO - login.py_doLogin:27 - Try Login, user=admin, pass=******
2023-01-28 22:44:20,368 [2482]   INFO - login.py_doLogin:39 - Logged in successfully, token=tok***
2023-01-28 22:44:20,369 [2483]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:44:43,883 [300]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:44:52,563 [291]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:44:54,754 [2482]   INFO - login.py_getUser:56 - Load User From File
2023-01-28 22:44:54,755 [2483]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:44:54,786 [2514]   INFO - monitor.py_getCameraList:13 - Get Camera List...
2023-01-28 22:44:54,805 [2533]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 22:44:54,856 [2584]   INFO - login.py_doLogin:27 - Try Login, user=admin, pass=******
2023-01-28 22:44:54,857 [2585]   INFO - login.py_doLogin:39 - Logged in successfully, token=tok***
2023-01-28 22:44:54,857 [2585]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:45:20,528 [290]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:45:22,620 [2382]   INFO - login.py_getUser:56 - Load User From File
2023-01-28 22:45:22,620 [2382] WARNING - user.py_getLoginStatus:42 - Auto login information not exist.
2023-01-28 22:45:22,646 [2408]   INFO - monitor.py_getCameraList:13 - Get Camera List...
2023-01-28 22:45:22,666 [2428]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 22:45:41,785 [294]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:45:59,599 [292]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:46:01,626 [2320]   INFO - login.py_getUser:55 - Load User From File
2023-01-28 22:46:01,711 [2404] WARNING - user.py_getLoginStatus:42 - Auto login information not exist.
2023-01-28 22:46:01,728 [2421]   INFO - monitor.py_getCameraList:13 - Get Camera List...
2023-01-28 22:46:01,748 [2441]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 22:46:12,011 [12704]   INFO - login.py_quit:17 - Exit Pressed.
2023-01-28 22:46:15,950 [290]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:46:18,064 [2405]   INFO - login.py_getUser:55 - Load User From File
2023-01-28 22:46:18,129 [2470] WARNING - user.py_getLoginStatus:42 - Auto login information not exist.
2023-01-28 22:46:18,148 [2489]   INFO - monitor.py_getCameraList:13 - Get Camera List...
2023-01-28 22:46:18,169 [2510]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 22:46:22,984 [7325]   INFO - login.py_doLogin:26 - Try Login, user=, pass=******
2023-01-28 22:46:22,984 [7325] WARNING - login.py_doLogin:47 - Login Failed, message: 用户名或密码错误
2023-01-28 22:46:22,985 [7326]   INFO - user.py_saveUser:29 - Saved User to file, username=
2023-01-28 22:46:24,766 [9107]   INFO - login.py_doLogin:26 - Try Login, user=, pass=******
2023-01-28 22:46:24,766 [9107] WARNING - login.py_doLogin:47 - Login Failed, message: 用户名或密码错误
2023-01-28 22:46:24,767 [9108]   INFO - user.py_saveUser:29 - Saved User to file, username=
2023-01-28 22:46:30,160 [14501]   INFO - login.py_doLogin:26 - Try Login, user=admin, pass=******
2023-01-28 22:46:30,160 [14501] WARNING - login.py_doLogin:47 - Login Failed, message: 用户名或密码错误
2023-01-28 22:46:30,161 [14502]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:46:33,685 [18025]   INFO - login.py_doLogin:26 - Try Login, user=admin, pass=******
2023-01-28 22:46:33,685 [18025] WARNING - login.py_doLogin:47 - Login Failed, message: 用户名或密码错误
2023-01-28 22:46:33,685 [18025]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:46:38,109 [22450]   INFO - login.py_doLogin:26 - Try Login, user=admin, pass=******
2023-01-28 22:46:38,109 [22450]   INFO - login.py_doLogin:38 - Logged in successfully, token=tok***
2023-01-28 22:46:38,110 [22451]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:48:02,787 [284]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:48:04,902 [2399]   INFO - login.py_getUser:55 - Load User From File
2023-01-28 22:48:04,968 [2465]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:48:04,990 [2487]   INFO - monitor.py_getCameraList:13 - Get Camera List...
2023-01-28 22:48:05,003 [2500]   INFO - login.py_doLogin:26 - Try Login, user=admin, pass=******
2023-01-28 22:48:05,003 [2500]   INFO - login.py_doLogin:38 - Logged in successfully, token=tok***
2023-01-28 22:48:05,003 [2500]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:48:05,010 [2507]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 22:48:40,277 [293]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:48:42,296 [2312]   INFO - login.py_getUser:60 - Load User From File
2023-01-28 22:48:42,301 [2317]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:48:42,320 [2336]   INFO - monitor.py_getCameraList:13 - Get Camera List...
2023-01-28 22:48:42,342 [2357]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 22:48:42,397 [2412]   INFO - login.py_doLogin:26 - Try Login, user=admin, pass=******
2023-01-28 22:48:42,399 [2414]   INFO - login.py_doLogin:38 - Logged in successfully, token=tok***
2023-01-28 22:48:42,400 [2415]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:48:51,887 [298]   INFO - __main__.py_<module>:13 - Program Loading.
2023-01-28 22:48:54,042 [2453]   INFO - login.py_getUser:61 - Load User From File
2023-01-28 22:48:54,094 [2505]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:48:54,110 [2521]   INFO - monitor.py_getCameraList:13 - Get Camera List...
2023-01-28 22:48:54,130 [2541]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
2023-01-28 22:48:54,138 [2549]   INFO - login.py_doLogin:26 - Try Login, user=admin, pass=******
2023-01-28 22:48:54,138 [2549]   INFO - login.py_doLogin:38 - Logged in successfully, token=tok***
2023-01-28 22:48:54,139 [2550]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:51:22,421 [292]   INFO - __main__.py_<module>:15 - Program Loading.
2023-01-28 22:51:24,450 [2321]   INFO - login.py_getUser:54 - Load User From File
2023-01-28 22:51:24,451 [2322]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:51:24,546 [2417]   INFO - login.py_doLogin:26 - Try Login, user=admin, pass=******
2023-01-28 22:51:24,546 [2417]   INFO - login.py_doLogin:38 - Logged in successfully, token=tok***
2023-01-28 22:51:24,547 [2418]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:53:00,379 [288]   INFO - __main__.py_<module>:22 - Program Loading.
2023-01-28 22:54:23,989 [293]   INFO - __main__.py_<module>:26 - Program Loading.
2023-01-28 22:54:40,507 [282]   INFO - __main__.py_<module>:25 - Program Loading.
2023-01-28 22:55:26,956 [286]   INFO - __main__.py_<module>:26 - Program Loading.
2023-01-28 22:56:24,839 [290]   INFO - __main__.py_<module>:14 - Program Loading.
2023-01-28 22:56:26,880 [2332]   INFO - login.py_getUser:54 - Load User From File
2023-01-28 22:56:26,880 [2332]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:56:26,978 [2430]   INFO - login.py_doLogin:26 - Try Login, user=admin, pass=******
2023-01-28 22:56:26,978 [2430]   INFO - login.py_doLogin:38 - Logged in successfully, token=tok***
2023-01-28 22:56:26,978 [2430]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:56:49,066 [309]   INFO - __main__.py_<module>:14 - Program Loading.
2023-01-28 22:56:51,111 [2353]   INFO - login.py_getUser:55 - Load User From File
2023-01-28 22:56:51,112 [2354]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:56:51,221 [2463]   INFO - login.py_doLogin:26 - Try Login, user=admin, pass=******
2023-01-28 22:56:51,221 [2463]   INFO - login.py_doLogin:38 - Logged in successfully, token=tok***
2023-01-28 22:56:51,222 [2464]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:56:51,242 [2484] WARNING - login.py_doLogin:47 - Login Failed, message: module 'webview.http' has no attribute 'running'
2023-01-28 22:56:51,243 [2485]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:57:11,387 [295]   INFO - __main__.py_<module>:14 - Program Loading.
2023-01-28 22:57:13,384 [2292]   INFO - login.py_getUser:55 - Load User From File
2023-01-28 22:57:13,385 [2293]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:57:13,482 [2390]   INFO - login.py_doLogin:26 - Try Login, user=admin, pass=******
2023-01-28 22:57:13,482 [2390]   INFO - login.py_doLogin:38 - Logged in successfully, token=tok***
2023-01-28 22:57:13,485 [2393]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:58:01,628 [291]   INFO - __main__.py_<module>:14 - Program Loading.
2023-01-28 22:58:03,666 [2329]   INFO - login.py_getUser:55 - Load User From File
2023-01-28 22:58:03,666 [2329]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:58:03,761 [2424]   INFO - login.py_doLogin:26 - Try Login, user=admin, pass=******
2023-01-28 22:58:03,761 [2424]   INFO - login.py_doLogin:38 - Logged in successfully, token=tok***
2023-01-28 22:58:03,762 [2425]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:58:15,171 [287]   INFO - __main__.py_<module>:14 - Program Loading.
2023-01-28 22:58:17,632 [2748]   INFO - login.py_getUser:55 - Load User From File
2023-01-28 22:58:17,632 [2748]   INFO - user.py_getLoginStatus:39 - Loaded User from file, username=admin
2023-01-28 22:58:17,732 [2848]   INFO - login.py_doLogin:26 - Try Login, user=admin, pass=******
2023-01-28 22:58:17,733 [2849]   INFO - login.py_doLogin:38 - Logged in successfully, token=tok***
2023-01-28 22:58:17,734 [2850]   INFO - user.py_saveUser:29 - Saved User to file, username=admin
2023-01-28 22:58:56,630 [41746]   INFO - monitor.py_getCameraList:13 - Get Camera List...
2023-01-28 22:58:56,652 [41768]   INFO - monitor.py_getCameraList:38 - Got Camera List:['USB2.0 HD UVC WebCam']
